src/main/java/net/starschema/clouddb/jdbc/BQSupportFuncts.java

```diff
@@ -507,47 +411,38 @@ public static String getTableIdFromTableDotGetId(String getId) {
      *                     <p>
      */
     public static List<Table> getTables(BQConnection connection,
-                                        String catalog, String schema, String tablename) throws IOException {
+                                        String projectIdFilter, String schema, String tablename) throws IOException {
```

Comment:
Not sure if this is a filter in the same sense anymore as it's doing an **exact** match.
It would be good to explore this as it does not seem 1:1 with the previous intention of the code
(not that the previous intention was right probably)